src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -1062,4 +1060,19 @@ public void onBuildingUpgradeComplete(AbstractBuilding building, int level)
     {
         return ServerUtils.getPlayersFromUUID(this.world, this.getPermissions().getMessagePlayers());
     }
+
+    public void triggerAchievement(Achievement achievement) {
+        if (this.colonyAchievements.contains(achievement)) {
+            return;
+        }
+
+        this.colonyAchievements.add(achievement);
+
+        AchievementUtils.syncAchievements(this);
+    }
+
+    public List<Achievement> getAchievements() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)